scripts/s3RobustCopy/main.py

```diff
@@ -83,10 +81,10 @@ def generateCypherKeyFromPassword(password, salt):
 def copyOrDownloadFile(
     downloadfolderlocal,
     key,
-    listOfSkipped,
-    listOfSuccess,
-    listOfFailed,
-    listOfFailedWithExceptions,
+    listOfSkipped_,
```

Comment:
what trailing `_`?

Reply:
this fixes the pylint issue that the variable is re-defined since there exists another one named `listOfSkipped` in the outer scope ;)